tools/publish/sync-secrets-to-winbuild.sh

```
@@ -0,0 +1,33 @@
+VM_PATH="C:/thecoin-env/secrets"
+
+SRC="${THECOIN_SECRETS:?THECOIN_SECRETS is not set on this host}"
+if [[ ! -d "$SRC" ]]; then
+  echo "THECOIN_SECRETS ($SRC) is not a directory" >&2
+  exit 1
+fi
+
+echo "Syncing $SRC -> $VM_USER@$VM_HOST:$VM_PATH"
+
+# (Re)create the target directory on the VM, so old/stale files don't linger
+ssh "$VM_USER@$VM_HOST" "powershell -NoProfile -Command \"Remove-Item -Recurse -Force $VM_PATH -ErrorAction SilentlyContinue; New-Item -ItemType Directory -Force -Path $VM_PATH | Out-Null\""
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)